assets/js/components/MercureApp.jsx

```diff
@@ -4,26 +4,36 @@ import httpBuildQuery from "../utils/httpBuildQuery";
 export class MercureApp extends React.Component {
     constructor(props) {
         super(props);
+        this.eventSource = null;
         this.state = {messages: [], newMessage: ''}
     }
 
     componentDidMount() {
-        const es = new EventSource('http://localhost:3000/.well-known/mercure?topic=' + encodeURIComponent('http://example.com/files/1'), {withCredentials: true});
+        this.eventSource = new EventSource(
```

Comment:
This is needed because otherwise the `es` variable could be garbaged collected when `componentDidMount` finishes and the SSE stream is immediately closed.